src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
```

Comment:
Consider using a constant for the daily cost value instead of hardcoding `40`. This will make the code more maintainable and the value can be easily changed in one place if needed.